index.js

```diff
@@ -21,28 +21,36 @@ function configureDefaults(options) {
   options.properties = options.properties || parsePropertiesFromEnv(process.env.PROPERTIES) || null;
   options.toConsole = !!options.toConsole;
   options.suiteTitleSeparedBy = options.suiteTitleSeparedBy || ' ';
+  options.rootSuiteTitle = 'Root Suite';
 
   return options;
 }
 
 function defaultSuiteTitle(suite) {
+  if (suite.root && suite.title === '') {
+      return this._options.rootSuiteTitle;
+  }
   return suite.title;
 }
 
-function fullSuiteTitle(suite, options) {
+function fullSuiteTitle(suite) {
   var parent = suite.parent;
   var title = [ suite.title ];
 
   while (parent) {
-    title.unshift(parent.title);
+    if (parent.root && parent.title === '') {
+      title.unshift(this._options.rootSuiteTitle);
+    } else {
+      title.unshift(parent.title);
+    }
     parent = parent.parent;
   }
 
-  return title.join(options.suiteTitleSeparedBy);
+  return title.join(this._options.suiteTitleSeparedBy);
 }
 
 function isInvalidSuite(suite) {
-  return suite.title === '' || suite.tests.length === 0 && suite.suites.length === 0;
+  return (!suite.root && suite.title === '') || suite.tests.length === 0 && suite.suites.length === 0;
```

Comment:
While you're touching this would you parenthesize the other half of the `||` for clarity